mantidproject/mantid
# Fix overplotting resetting axes scales

**Description of work.**
This PR fixes a bug where changing the axes scales on a plot and overplotting a curve would cause the axes to reset to their default scale.

**To test:**
Plot some spectra.
Change the scales of the axes, either by double-clicking on the axes or in the figure options.
Drag a workspace onto a plot to overplot some spectra.
Check that the axes scales have stayed the same.

Fixes #28611 

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
